### PR TITLE
build(deps): Bump axios version to mitigate vulnerability

### PR DESCRIPTION
**Pull request by adayNU**
Resolves https://github.com/advisories/GHSA-cph5-m8f7-6c5x

### CODE REVIEW

**Comment by adayNU** [comment]
@JoaoPedroAS51 FYI

File: yarn.lock [hunk 1]
```diff
@@ -3220,6 +3220,13 @@ axios@^0.21.1:
   dependencies:
     follow-redirects "^1.10.0"
 
+axios@^0.21.2:
```
Comment (bmulholland) [inline yarn.lock:3223]: Something seems off - why is this included when it wasn't before?

**Comment by bmulholland** [comment]
Fixed in https://github.com/nuxt-community/auth-module/pull/1372
